packages/destination-actions/src/destinations/loops/sendEvent/index.ts

```diff
@@ -0,0 +1,55 @@
+import type { ActionDefinition } from '@segment/actions-core'
+import type { Settings } from '../generated-types'
+import type { Payload } from './generated-types'
+
+const action: ActionDefinition<Settings, Payload> = {
+  title: 'Send Event',
+  description: 'Send an event for a contact in Loops',
+  defaultSubscription: 'type = "track"',
+
+  fields: {
+    email: {
+      label: 'Contact Email',
+      description: 'Email address for the contact.',
+      type: 'string',
+      format: 'email',
+      required: false,
+      default: {
+        '@if': {
+          exists: { '@path': '$.traits.email' },
+          then: { '@path': '$.traits.email' },
+          else: { '@path': '$.email' }
+        }
+      }
```

Comment:
Hi @askkaz , for a track() event you need to check these 2 locations for the email address. See below. 

```suggestion
      default: {
        '@if': {
          exists: { '@path': '$.properties.email' },
          then: { '@path': '$.properties.email' },
          else: { '@path': '$.context.traits.email' }
        }
      }
```